script/events/room.js

```diff
@@ -49,7 +49,37 @@ Events.Room = [
 				}
 			}
 		}
-	}, { /* Noises Outside  --  gain wood/fur */
+	}, 
+	{
+		title: _('Fire'),
+		isAvailable: function() { 
+			return $SM.get('game.buildings["hut"]', true) > 0 && $SM.get('game.population', true) > 5;
+		},
+		scenes: {
+			'start': {
+				text: [
+					_('A fire rampages through one of your huts, destroying it.'),
+					_('Saddly, all residents in the hut perished in the fire.')

```

Comment:
I would probably take "sadly" out. ADR is best when it's stark and unforgiving. It's not sad, it's life.